### PR TITLE
Implement media usage for non-translatable blocks

### PR DESCRIPTION
**Pull request by strategio**
Please check also the same approach on https://github.com/OnTheGoSystems/wpml-page-builders-beaver-builder/pull/14.

https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-5834

### CODE REVIEW

**Comment by strategio** [comment]
@OnTheGoSystems/wpml please review.

File: src/media/class-wpml-elementor-update-media-factory.php [hunk 1]
```diff
@@ -2,13 +2,33 @@
...
+			),
+			new WPML_Page_Builders_Media_Usage( $this->get_media_translate(), new WPML_Media_Usage_Factory() )
 		);
 	}
+
+	/** @return WPML_Page_Builders_Media_Translate */
+	private function get_media_translate() {
+		global $sitepress;
+
+		if ( ! $this->media_translate ) {
```
Comment (brucepearson) [inline src/media/class-wpml-elementor-update-media-factory.php:25]: @strategio Same comment here that I made for Beaver Builder.

**Review by brucepearson — APPROVED** [review]
@strategio This looks ok. +1
Please check the cached variable.

File: src/media/class-wpml-elementor-update-media-factory.php (hunk 1 above)
Comment (strategio) [reply src/media/class-wpml-elementor-update-media-factory.php:25]: Same response @brucepearson, I want to emphasize the need for a single instance of media translate in this factory.

I merged the PR so I can progress on the next task.

**Pull request merged**